### ISSUE TITLE
Consider using group only if scrollable

### ISSUE THREAD

**Issue by mcking65** [Heydon/react-sortable-table-demo#7]
The group wrapper on the table creates undesirable screen reader verbosity. I understand the need when there is a focusable div wrapping the table to support scrolling. However, if the table is not scrollable and the div is thus not focusable, it would make a cleaner presentation if the role group and aria-labelledby were left out.
